src/main/java/com/feelmycode/parabole/service/EventService.java

```diff
@@ -117,6 +123,28 @@ public List<Event> getEventsBySellerId(Long userId) {
         return eventRepository.findAllBySellerAndIsDeleted(seller, false);
     }
 
+    /**
+     * 검색 조건으로 이벤트 목록 조회 (경품 목록 제외)
+     */
+    public List<EventSearchResponseDto> getEventsSearch(EventSearchRequestDto eventSearchRequestDto) {
+
+        List<Event> eventList = null;
+        if(!StringUtils.isEmpty(eventSearchRequestDto.getEventType())) {
+            eventList = eventRepository.findAllByTypeAndIsDeleted(eventSearchRequestDto.getEventType().getCode(), false);
+        } else if (!StringUtils.isEmpty(eventSearchRequestDto.getEventTitle())) {
+            eventList = eventRepository.findAllByTitleContainingAndIsDeleted(eventSearchRequestDto.getEventTitle(), false);
+        } else if (!StringUtils.isEmpty(eventSearchRequestDto.getDateDiv())) {
+            eventList = eventSearchRequestDto.getDateDiv() < 1 ?
+                eventRepository.findAllByStartAtBetweenAndIsDeleted(eventSearchRequestDto.getFromDateTime(), eventSearchRequestDto.getToDateTime(), false) : eventRepository.findAllByEndAtBetweenAndIsDeleted(eventSearchRequestDto.getFromDateTime(), eventSearchRequestDto.getToDateTime(), false);
```

Comment:
이 쪽이 조금 더 코드를 한 눈에 보기 좋을 거 같아요
```suggestion
            eventList = eventSearchRequestDto.getDateDiv() < 1 
            ? eventRepository.findAllByStartAtBetweenAndIsDeleted(eventSearchRequestDto.getFromDateTime(), eventSearchRequestDto.getToDateTime(), false) 
            : eventRepository.findAllByEndAtBetweenAndIsDeleted(eventSearchRequestDto.getFromDateTime(), eventSearchRequestDto.getToDateTime(), false);
```